### PR TITLE
SPEC-52 Upgrade JPA to Jakarta 2.2.2

### PR DESCRIPTION
**Pull request by Cousjava**

### CODE REVIEW

**Comment by Cousjava** [comment]
Jenkins test please

File: appserver/tests/paas/basic-jpa/pom.xml [hunk 1]
```diff
@@ -93,7 +93,7 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-      	    <groupId>javax.persistence</groupId>
+      	    <groupId>jakarta.persistence</groupId>
             <artifactId>persistence-api</artifactId>
```
Comment (arjantijms) [inline appserver/tests/paas/basic-jpa/pom.xml]: I think this one doesn't exist, see:

https://mvnrepository.com/artifact/jakarta.persistence

File: appserver/tests/paas/basic-shared-service-test/pom.xml [hunk 2]
```diff
@@ -93,7 +93,7 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-      	    <groupId>javax.persistence</groupId>
+      	    <groupId>jakarta.persistence</groupId>
```
Comment (arjantijms) [inline appserver/tests/paas/basic-shared-service-test/pom.xml]: I think this one doesn't exist, see:

https://mvnrepository.com/artifact/jakarta.persistence

File: appserver/tests/paas/ext-shared-service-enable-disable-test/pom.xml [hunk 3]
```diff
@@ -93,7 +93,7 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-      	    <groupId>javax.persistence</groupId>
+      	    <groupId>jakarta.persistence</groupId>
```
Comment (arjantijms) [inline appserver/tests/paas/ext-shared-service-enable-disable-test/pom.xml]: I think this one doesn't exist, see:

https://mvnrepository.com/artifact/jakarta.persistence

File: appserver/tests/paas/external-and-shared-service/pom.xml [hunk 4]
```diff
@@ -93,7 +93,7 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-      	    <groupId>javax.persistence</groupId>
+      	    <groupId>jakarta.persistence</groupId>
```
Comment (arjantijms) [inline appserver/tests/paas/external-and-shared-service/pom.xml]: I think this one doesn't exist, see:

https://mvnrepository.com/artifact/jakarta.persistence

File: appserver/tests/paas/lazy-shared-service-test/pom.xml [hunk 5]
```diff
@@ -93,7 +93,7 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-      	    <groupId>javax.persistence</groupId>
+      	    <groupId>jakarta.persistence</groupId>
```
Comment (arjantijms) [inline appserver/tests/paas/lazy-shared-service-test/pom.xml]: I think this one doesn't exist, see:

https://mvnrepository.com/artifact/jakarta.persistence

File: appserver/tests/paas/multiple-spe-test/spetest/pom.xml [hunk 6]
```diff
@@ -109,7 +109,7 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>javax.persistence</groupId>
+            <groupId>jakarta.persistence</groupId>
```
Comment (arjantijms) [inline appserver/tests/paas/multiple-spe-test/spetest/pom.xml]: I think this one doesn't exist, see:

https://mvnrepository.com/artifact/jakarta.persistence

File: appserver/tests/paas/basic-jpa/pom.xml (hunk 1 above)
Comment (Cousjava) [reply appserver/tests/paas/basic-jpa/pom.xml]: yes it does

**Comment by Cousjava** [comment]
Jenkins test please

File: appserver/tests/paas/basic-jpa/pom.xml (hunk 1 above)
Comment (arjantijms) [reply appserver/tests/paas/basic-jpa/pom.xml]: Are you sure? Looking at Maven central directly, I only see a folder called "jakarta.persistence-api":

https://repo1.maven.org/maven2/jakarta/persistence/

Maven also gives the following error:

> Failure to find jakarta.persistence:persistence-api:jar:2.2.2 in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]

Comment (jGauravGupta) [reply appserver/tests/paas/basic-jpa/pom.xml]: I also checked only `jakarta.persistence-api` artifacts exist.

**Comment by Cousjava** [comment]
Jenkins test please

**Comment by Cousjava** [comment]
Jenkins test please

**Pull request merged**